sdk/storage/storage-blob/src/index.ts

```diff
@@ -48,3 +48,6 @@ export {
   BlobBeginCopyFromUrlPollState,
   CopyPollerBlobClient
 } from "./pollers/BlobStartCopyFromUrlPoller";
+
+import { AvroReadable } from '../../storage-internal-avro/src'
```

Comment:
You can use single statement `export { AvroReadable } from ""`

Reply:
Thanks for the tip. 
And these two lines are only for test purpose. We probably won't export `AvroReadable ` in the end.

Reply:
Removed.